app/services/ImagePickerService.js

```diff
@@ -0,0 +1,23 @@
+import ImagePicker from 'react-native-image-picker';
+
+export const selectPhotoTapped = () => (
```

Comment:
oh, so you want to separate button(Select Image) and image/placeholder?
if so i don't think it's useful. `FormImagePicker` is quite re-usable.